src/commands/function-flush.json

```diff
@@ -0,0 +1,37 @@
+{
+    "FLUSH": {
+        "summary": "PATCH__TBD__29__",
+        "complexity": "PATCH__TBD__28__",
+        "group": "scripting",
+        "since": "7.0.0",
+        "arity": -2,
+        "container": "FUNCTION",
+        "function": "functionFlushCommand",
+        "command_flags": [
+            "NOSCRIPT",
+            "MAY_REPLICATE"
```

Comment:
we'll probably decide to change `MAY_REPLICATE` to `WRITE` soon.
it can be done in the other PR, if this one is merged first.
just noting that if both PRs are pending, and later merged separately, we can forget.

Reply:
I agree, will set it as write commands here and will add tests that verifies.

Reply:
since i think this PR is gonna be merge first, we can leave it as is here.
and then make sure the other PR with an explicit separate topic), handles that for all.
just need to be sure to handle FLUSH in that other PR, in case you draft it before this one is merged.

Reply:
#9953